parsec/mca/device/device.c

```diff
@@ -375,7 +375,6 @@ void parsec_devices_print_statistics(parsec_context_t *parsec_context, uint64_t
     float best_required_in, best_required_out;
     char *data_in_unit, *data_out_unit, *d2d_unit;
     char *required_in_unit, *required_out_unit;
-    char percent1[64], percent2[64], percent3[64];
```

Comment:
What's wrong with printing `nan` when there are nothing to be printed ?

Reply:
That also answer the next question, dividing by 0 will naturally produce a nan that will be printed without all that complication.

Reply:
or raise a signal that we don't deal with. I would rather handle the `nan` internally than rely on an external mechanism that we do not control.

Reply:
You are correct the changed code is valid only if the implementation is IEEE754. `If the second operand is zero, the behavior is undefined, except that if the IEEE floating-point arithmetic is supported, and the floating-point division is taking place, then`. 

That same defect is present in the code per-device computation right above,  I will update it to match.